packages/app/src/elements/settings-security.ts

```diff
@@ -625,13 +625,15 @@ export class SettingsSecurity extends StateMixin(Routing(LitElement)) {
         `;
     }
 
-    private _renderBiometricUnlockCurrentDevice(currentDevice: DeviceInfo, currentAuthenticator?: AuthenticatorInfo) {
-        const supportsPlatformAuth = supportsPlatformAuthenticator();
+    private async _renderBiometricUnlockCurrentDevice(
```

Comment:
I had to rewrite this a bit in order to get the proper value from `supportsPlatformAuthenticator()` because `until(!!currentAuthenticator || supportsPlatformAuth, true)` wasn't being easy to deal with, and I noticed it was also saying a device was supported when it wasn't, etc.